config/routes.rb

```diff
@@ -62,6 +62,15 @@
 
   resources :teams, except: :show
 
+  resources :administrations, only: :index
```

Comment:
better use a singular resource here, see http://guides.rubyonrails.org/routing.html#singular-resources
`resource :administration, only: :show`
* the route will have a better wording: `/administration` 
* controller name, view paths, etc. stay the same